controllers/movie.controller.js

```diff
@@ -4,12 +4,25 @@ class MovieController {
         this.movieRepository = movieRepository;
     }
 
-    getAll() {
-        return this.movieRepository.find();
+    formatMovie(movie) {
+        return {
+            id: movie._id,
+            genre: movie.genre,
+            posterUrl: movie.posterUrl,
+            rating: movie.rating,
+            title: movie.title,
+            contributorId: movie.contributorId
+        }
     }
```

Comment:
Might have forgotten the purpose of formatMovie..

Reply:
It's used to convert the document from MongoDB into a JS object which can be more friendly to use eg. map `_id` to `id`.